test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetListPackageTests.cs

```diff
@@ -198,7 +198,7 @@ public async Task DotnetListPackage_InvalidFramework_Fail()
             }
         }
 
-        [Fact(Skip = "Requires a version of dotnet.exe that supports forwarding the --deprecated command argument. https://github.com/NuGet/Home/issues/8440")]
+        [PlatformFact(Platform.Windows)]
```

Comment:
Why is this now Windows-only?

Reply:
It was always supposed to be Windows only. 
The patching infrastructure is not set up on linux/mac.